schema.sql

```diff
@@ -1,4 +1,20 @@
 /* Database schema to keep the structure of entire database. */
+--milestone_4
+
+CREATE TABLE vets (
+    id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    name VARCHAR(60),
+    age INT,
+    date_of_graduation DATE
+)
+
+CREATE TABLE visits (
+    visit_date DATE,
+    animals_id INT REFERENCES animals(id),
+    vets_id    INT REFERENCES vets(id)
+);
+
+
```

Comment:
- [ ] Kindly make sure you also add a new join table with the name **specializations** as required for **[the project](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md)**.
![Screenshot from 2022-09-14 02-44-30](https://user-images.githubusercontent.com/83042742/190033915-bafd625f-9ea5-45e2-aae3-8a9c4a6f3f9c.png)